src/veins/modules/mobility/traci/TraCIScenarioManager.h

```diff
@@ -63,11 +63,11 @@ class TraCICommandInterface;
  */
 class TraCIScenarioManager : public cSimpleModule {
 public:
-    static const std::string TRACI_INITIALIZED_SIGNAL_NAME;
-    static const std::string TRACI_MODULE_ADDED_SIGNAL_NAME;
-    static const std::string TRACI_MODULE_REMOVED_SIGNAL_NAME;
-    static const std::string TRACI_TIMESTEP_BEGIN_SIGNAL_NAME;
-    static const std::string TRACI_TIMESTEP_END_SIGNAL_NAME;
+    static const simsignal_t traciInitializedSignal;
+    static const simsignal_t traciModuleAddedSignal;
+    static const simsignal_t traciModuleRemovedSignal;
+    static const simsignal_t traciTimestepBeginSignal;
+    static const simsignal_t traciTimestepEndSignal;
```

Comment:
Are `static const simsignal_t`s really the way to go? If so, cool!
Previously I just used them as default-initialized private members. That also keeps them out of constructor lists and would avoid any problems coming along with external linkage (the typical issues of const data members). Plus: For a pseudo-singleton like the `TraciScenarioManager`, it should not be a performance problem at all.
Though that way would require all using classes to keep a copy of the `simsignal_t` of their own... so maybe the `ststic const` way is the best. I just am not sure about all the implications.